.evergreen/.evg.yml

```diff
@@ -2180,8 +2168,8 @@ buildvariants:
     - name: "csfle-tests-with-mongocryptd"
 
 - matrix_name: "socks5-tests"
-  matrix_spec: { os: "linux", ssl: ["nossl", "ssl"], version: [ "latest" ], topology: ["replicaset"] }
-  display_name: "Socks5: ${version} ${topology} ${ssl} ${jdk} ${os}"
+  matrix_spec: { os: "linux", ssl: ["nossl", "ssl"], version: [ "latest" ], topology: ["replicaset"], socks_auth: ["auth", "noauth"] }
+  display_name: "${socks_auth} SOCKS5 proxy: ${version} ${topology} ${ssl} ${jdk} ${os}"
```

Comment:
Let's move `socks_auth` after `SOCKS5 proxy` so that they sort together in the display